File: forJavaLabs/Lab3/FilmStudio/src/FilmStudio/Worker.java
package FilmStudio;

import model.enums.FilmGenre;



public class Worker {
private String occupationName;
private String firstName;
private String lastName;
private int age;
private int salaryPerHour;
private FilmGenre genre;
private int yearExperience;

public String getOccupationName() {
	return occupationName;
}


public void setOccupationName(String occupationName) {
	this.occupationName = occupationName;
}


public FilmGenre getGenre() {
	return genre;
}


public Worker() {
}


public Worker(String occupationName, String firstName, String lastName, int age, int salaryPerHour,
		 FilmGenre genre, int yearExperience) {
	this.firstName = firstName;
	this.lastName = lastName;
	this.age = age;
	this.salaryPerHour = salaryPerHour;
	this.genre = genre;
	this.yearExperience = yearExperience;
	this.occupationName = occupationName;
}


public String getFirstName() {
	return firstName;
}


public void setFirstName(String firstName) {
	this.firstName = firstName;
}


public String getLastName() {
	return lastName;
}


public void setLastName(String lastName) {
	this.lastName = lastName;
}


public int getAge() {
	return age;
}


public void setAge(int age) {
	this.age = age;
}


public int getSalaryPerHour() {
	return salaryPerHour;
}


public void setSalaryPerHour(int salaryPerHour) {
	this.salaryPerHour = salaryPerHour;
}


public int getYearExperience() {
	return yearExperience;
}


public void setYearExperience(int yearExperience) {
	this.yearExperience = yearExperience;
}
@Override
public String toString() {
	return "Worker [occupation = "+ occupationName  + ", first Name = " + firstName + ", last Name = " + lastName + ",age = "
			+ age + ", salary per hour = " + salaryPerHour + ",film genre can play = " + genre + ", year Experience = " + yearExperience + "]"+"\n";
}
}

File: forPythonLabs/Lab2/Studio/Worker.py
from enums.FilmGenre import FilmGenre


class Worker:
    occupation_name, first_name, last_name, age, salary_per_hour, year_experience, film_genre = "", "", "", 0, 0, \
                                                                                                0, None

    def __init__(self, occupation_name="", first_name="", last_name="", age=0,
                 salary_per_hour=0, year_experience=0, film_genre=FilmGenre.OTHER):
        self.first_name = first_name
        self.last_name = last_name
        self.age = age
        self.salary_per_hour = salary_per_hour
        self.year_experience = year_experience
        self.film_genre = film_genre
        self.occupation_name = occupation_name

    def reset_values(self, occupation_name, first_name, last_name, age, salary_per_hour, year_experience,
                     film_genre):
        self.occupation_name = occupation_name
        self.last_name = last_name
        self.first_name = first_name
        self.age = age
        self.salary_per_hour = salary_per_hour
        self.year_experience = year_experience
        self.film_genre = film_genre

    def to_string(self):
        print("Occupation Name: " + self.occupation_name + ", First Name: " + self.first_name +
              ", Last Name: " + self.last_name + ", age: " + str(self.age) + ", Salary Per Hour : "
              + str(self.salary_per_hour) + ", Year Experience: " + str(self.year_experience) + ", Film Genre"
              + str(self.film_genre))
